2020-01-12 16:47:55 can only concatenate str (not "tuple") to str
2020-01-12 16:49:04 can only concatenate str (not "tuple") to str
2020-01-12 16:58:31 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 240, in <module>
    categories = categories)
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 108, in __init__
    self.GenerateWikiText()
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 142, in GenerateWikiText
    """ + self.categories + """
TypeError: can only concatenate str (not "tuple") to str
2020-01-12 17:00:19 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 240, in <module>
    categories = categories)
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 108, in __init__
    self.GenerateWikiText()
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 142, in GenerateWikiText
    """ + str(self.categories) + """
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 20:28:16 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 240, in <module>
    categories = categories)
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 108, in __init__
    self.GenerateWikiText()
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 142, in GenerateWikiText
    """ + str(self.categories) + """
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 20:30:30 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 240, in <module>
    categories = categories)
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 108, in __init__
    self.GenerateWikiText()
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/commons.py", line 142, in GenerateWikiText
    """ + str(self.categories) + """
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 20:40:55 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 234, in <module>
    source = 'https://collection.smk.dk/#/en/detail/'+accession_number,
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 20:46:51 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 234, in <module>
    source = 'https://collection.smk.dk/#/en/detail/'+accession_number,
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 21:58:27 can only concatenate str (not "tuple") to str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 234, in <module>
    source = 'https://collection.smk.dk/#/en/detail/'+accession_number,
TypeError: can only concatenate str (not "tuple") to str
2020-01-13 22:15:51 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 271, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:17:45 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 271, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:25:26 'list' object has no attribute 'get'
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 194, in <module>
    smk_creator=str(item['production'].get['creator'])
AttributeError: 'list' object has no attribute 'get'
2020-01-13 22:26:42 list indices must be integers or slices, not str
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 194, in <module>
    smk_creator=str(item['production']['creator'])
TypeError: list indices must be integers or slices, not str
2020-01-13 22:27:57 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:27:57 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:01 'creator'
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 194, in <module>
    smk_creator=str(item['production'][0]['creator'])
KeyError: 'creator'
2020-01-13 22:28:02 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:05 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:05 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:09 'unit'
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 203, in <module>
    print(dimension['unit'])
KeyError: 'unit'
2020-01-13 22:28:09 'type'
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 201, in <module>
    print(dimension['type'])
KeyError: 'type'
2020-01-13 22:28:09 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:14 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:14 'creator'
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 194, in <module>
    smk_creator=str(item['production'][0]['creator'])
KeyError: 'creator'
2020-01-13 22:28:16 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:16 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:17 list index out of range
Traceback (most recent call last):
  File "/Users/kimbach/Documents/wikilabskultur/wikilabskultur/smkbatch.py", line 272, in <module>
    smk_creator=creator[1]+' '+creator[0]
IndexError: list index out of range
2020-01-13 22:28:18 Closing network session.
2020-01-13 22:28:18 Network session closed.
